nym-wallet/src/theme/theme.tsx

```diff
@@ -303,6 +303,24 @@ export const getDesignTokens = (mode: PaletteMode): ThemeOptions => {
           },
         },
       },
+      MuiSelect: {
+        defaultProps: {
+          MenuProps: {
+            PaperProps: {
+              sx: {
+                '&& .Mui-selected': {
+                  color: nymPalette.highlight,
+                  backgroundColor: (t) =>
+                    t.palette.mode === 'dark' ? `${t.palette.background.default} !important` : '#FFFFFF !important',
+                },
+                '&& .Mui-selected:hover': {
+                  backgroundColor: 'rgba(251, 110, 78, 0.08) !important',
+                },
+              },
+            },
+          },
+        },
+      },
```

Comment:
I'm not sure how I feel about adding these global styles. I think the best solution is to create our own `StyledSelect` component with this styling BUT perhaps  leave it like this for now if it works, and if we need to change it later we will